src/main/java/mate/jdbc/service/ManufacturerServiceImpl.java

```diff
@@ -0,0 +1,41 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import mate.jdbc.dao.ManufacturerDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Manufacturer;
+
+@Service
+public class ManufacturerServiceImpl implements ManufacturerService {
+    @Inject
+    private ManufacturerDao manufacturerDao;
+
+    @Override
+    public Manufacturer create(Manufacturer manufacturer) {
+        return manufacturerDao.create(manufacturer);
+    }
+
+    @Override
+    public Manufacturer get(Long id) {
+        if (manufacturerDao.get(id).isEmpty()) {
+            throw new RuntimeException("Can't find Manufacturer by id " + id);
+        }
+        return manufacturerDao.get(id).get();
```

Comment:
You are going to DB 2 times??
```suggestion
        return manufacturerDao.get(id)
                .orElseThrow(() -> new NoSuchElementException("Can't find Manufacturer by id " + id));
```